### ISSUE TITLE
Got error for query schema "Introspection is not allowed for the current request"

### ISSUE THREAD

**Issue by JerryWang20** [ChilliCream/graphql-platform#7473]
### Product

Hot Chocolate

### Version

14.0.0 rc1

### Link to minimal reproduction

https://github.com/ChilliCream/graphql-platform

### Steps to reproduce

The function works well in locala VS, after deploy to Azure. open the schema with error "Introspection is not allowed for the current request". But i already setting following in the code. Pls help on how to fix this
builder.Services.AddGraphQLServer()
    .AllowIntrospection(true)

### What is expected?

Should open the graphql schema

### What is actually happening?

The function works well in locala VS, after deploy to Azure. open the schema with error "Introspection is not allowed for the current request". But i already setting following in the code. Pls help on how to fix this
builder.Services.AddGraphQLServer()
    .AllowIntrospection(true)

**Comment by michaelstaib** [comment]
This is the expected behavior ... we by default secure the GraphQL server .... however ... the new bcp should pick up the schema file with rc2 and use this instead of introspection ...

**Comment by JerryWang20** [comment]
if we want to expose the schema, how to enable it?

**Comment by JerryWang20** [comment]
i install bcp and try to open the schema with same error return

**Comment by JerryWang20** [comment]
@michaelstaib  any document describe the steps?

**Comment by glen-84** [comment]
@JerryWang20

```cs
builder.Services
    .AddGraphQLServer()
    .RemoveIntrospectionAllowedRule();
```

**Comment by AndrzejKroczynski** [comment]
Is there any way to enable that?

In this article https://chillicream.com/blog/2024/08/30/hot-chocolate-14 I see following options:

```
builder
    .AddGraphQLServer()
    .ModifyRequestOptions(o => o.EnableIntrospection = true);
```

But it is not available on version 14.0.0

**Comment by glen-84** [comment]
@AndrzejKroczynski https://github.com/ChilliCream/graphql-platform/pull/7637.
